### ISSUE TITLE
Scroll doesn't work using TouchPad. Works Okay with Mouse device

### ISSUE THREAD

**Issue by coach1990** [FlutterFlow/flutterflow-issues#3516]
### Can we access your project?

- [ ] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I designed a page  with Column Widget, which has "Scrollable' property set to 'Allow'. When I clcik 'Preview App' on my Laptop, the Page won't Scroll if I use TouchPad. However, if I use Mouse device then I am able to Scroll the Page.

### Expected Behavior

Page Scroll should work using TouchPad also.

### Steps to Reproduce

Add a Page
Add a Column - Change 'Scrolleable' Property to Allow Scrolling
Add 4-5 Carousel widget so whole page is filled and, we could scroll to see addition Carousel widgets.
Click Preview App
Try to Scroll using Laptop TouchPad (page won't scroll)
Try to Scroll after connecting Mouse device (Scrolling works)

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4sls/l6otPvsdG1ZDuc8dB/TkQJUw8U508scp/bxIgGIDzPOw+ZPXsUFFWTc+5dAk6e0Smml4C0NLukYXuGPI5C0idUa5Jzrh1WzvvVmehUZOBDqiOdEdBM9tUC2GM5pjZkxZ5HLFucl4s7U6yG+/KUXTrHu/AIndISq/LZO4=

### Visual documentation

https://github.com/user-attachments/assets/00dd24ff-88f4-48f7-aa01-16350b7aadeb

### Environment

```markdown
- FlutterFlow version: FlutterFlow v4.1.79+ released July 31, 2024 Flutter version is 3.22.2
- Platform: Desktop App Windows
- Browser name and version: Microsoft Edge Version 127.0.2651.74 (Official build) (64-bit)
- Operating system and version affected: Windows 11 Home Single Language , 23H2
```

### Additional Information

Actually the issue is, I use Touchpad a Lot, but i have to use Mouse for this purpose only, to check if Scrol is working ? I hope its not too much trouble for you. Thanks for your time and efforts on this.

**Comment by Alezanello** [comment]
Hello!

I wasn't able to reproduce this issue. The scrolling on the preview page is working fine using both the laptop touchpad and a mouse. I filmed myself showing that I tried both methods on the laptop.

[20240731_180002.mp4](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/17ec75e1-c6d1-434d-8f2b-6a4771aa33a4/6c7bf5d0-5ca8-4d2d-8760-4e7862dfbae5?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS8xN2VjNzVlMS1jNmQxLTQzNGQtOGYyYi02YTQ3NzFhYTMzYTQvNmM3YmY1ZDAtNWNhOC00ZDJkLTg3NjAtNGU3ODYyZGZiYWU1IiwiaWF0IjoxNzIyNDU5ODM1LCJleHAiOjMzMjkzMDE5ODM1fQ.xTCvXc4sgeGSJy8jfUO7oQTiMgJcgKxnte2bX-d-ZDA)

<img src="https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/aacf3b1f-b269-48e8-8bd8-e61081cf2665/1627eae3-c1e8-4f82-a2f3-7be3dfb32989?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9hYWNmM2IxZi1iMjY5LTQ4ZTgtOGJkOC1lNjEwODFjZjI2NjUvMTYyN2VhZTMtYzFlOC00ZjgyLWEyZjMtN2JlM2RmYjMyOTg5IiwiaWF0IjoxNzIyNDU5ODM1LCJleHAiOjMzMjkzMDE5ODM1fQ.Izy2J0QTjUlw2gQ6eR6PpBVX5nxh_xbtgOBZxBuNYQQ " alt="image.png" width="1856" height="738" />

Best regards,<br>Azanello
